test/spec/services/favorites.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+describe('Service: Favorites', function () {
+
+  // load the service's module
+  beforeEach(module('publicacionesFavoritasApp'));
+
+  // instantiate service
+  var Favorites;
+  beforeEach(inject(function (favorites) {
+    Favorites = favorites;
+  }));
+
+  it('should be favourite', function () {
+    Favorites.update('MLA1234');
+    expect(Favorites.isFavourite('MLA1234')).toBe(true);
+  });
+
+  it('should not be favourite', function () {
+    Favorites.update('MLA1235');
+    Favorites.update('MLA1235');
+    expect(Favorites.isFavourite('MLA1235')).toBe(false);
+  });
+
+  it('should not be favourite because this element exists before as a favourite', function () {
+    Favorites.update('MLA1234');
```

Comment:
Está bueno tratar que los test unitarios no tengan dependencia entre sí. En este caso si eliminases el test `should be favorite` este test dejaría de pasar.